quartz/job.go

```diff
@@ -153,6 +203,20 @@ func (cu *CurlJob) Key() int {
 	return HashCode(cu.description)
 }
 
+// Response returns the response of the CurlJob.
+func (cu *CurlJob) Response() *http.Response {
+	cu.RLock()
+	defer cu.RUnlock()
+	return cu.response
```

Comment:
the response here is still mutable: the mutex only prevents it from being set or unset when the function returns... another actor could modify the repose itself (in  theory.)

Reply:
This http.reponse is special, for example, after we read the http body, we can no read it again.

I think we can ignore this problem. 😁

Reply:
Yes, the getter method have been added to eliminate data race during concurrent read/write operations.